Tests/Microsoft.AppCenter.Test.WindowsDesktop.Shared/Ingestion/Http/HttpNetworkAdapterTest.cs

```diff
@@ -10,17 +10,15 @@ public class HttpNetworkAdapterTest
         public void EnableTls12WhenDisabled()
         {
             ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11;
-            HttpNetworkAdapter.EnableTls12();
```

Comment:
I think we need to fix this test. Can we setup compile const `NET461` in test mode?